Pipfile.lock

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
I think we'd want a `Pipfile`, but not a `Pipfile.lock`

Reply:
Got it!